app.py

```diff
@@ -43,6 +43,10 @@ def ratelimit_handler(e):
 def index():
     return render_template('index.html', current_semester = current_semester) 
 
+
+import pandas as pd
+courses = pd.read_csv('data/courses-2024.csv')
```

Comment:
can rename to more specific variable name related to aspc mainsite, so it's clearer where this data is from and how it differs from our other courses data